src/schema/rules/tabular_data/task.yaml

```diff
@@ -10,6 +10,11 @@ TaskEvents:
     response_time: optional
     HED: optional
     stim_file: optional
+    channel:
+      level: optional
+      description_addendum: |
+        Note that this column only applies to data types where
+        channels are specified, such as EEG, iEEG, MEG or NIRS.
```

Comment:
Technically also motion, no?
So maybe better than listing all the relevant data types maybe just say:
`EEG, iEEG, MEG...` ?

Reply:
I think the "such as" serves as a "for example"

Reply:
ha yeah it can be read that way too: fine then. :-)